docs/src/examples/components/List/Types/ListExampleSelection.tsx

```diff
@@ -1,30 +1,28 @@
 import React from 'react'
 import { List, Image } from '@stardust-ui/react'
 
-const selection = knobs => (knobs === undefined ? true : knobs.selection)
-
-const ListExampleSelection = ({ knobs }) => (
-  <List selection={selection(knobs)}>
+const ListExampleSelection = () => (
+  <List selection={true}>
     <List.Item
       media={<Image src="public/images/avatar/small/matt.jpg" avatar />}
       header="Irving Kuhic"
       headerMedia="7:26:56 AM"
       content="Program the sensor to the SAS alarm through the haptic SQL card!"
-      selection={selection(knobs)}
+      selection={true}
```

Comment:
nit: the `={true}` portion is not required.  A prop alone means `true`.  We'll add an auto format fix for this but let's remove it from here for now.

Reply:
yep, right